src/updater/updater.cpp

```diff
@@ -413,20 +439,128 @@ updateTurnPenalties(const UpdaterConfig &config,
 
         if (turn_weight_penalty < 0)
         {
-            util::Log(logWARNING) << "Negative turn penalty at " << turn_index.from_id << ", "
-                                  << turn_index.via_id << ", " << turn_index.to_id
+            util::Log(logWARNING) << "Negative turn penalty at " << internal_turn.from_id << ", "
```

Comment:
You could use `osm_turn` instead of `internal_turn` here because it is easier for the user to see OSM ids instead of internal IDs.